src/api/writer-generator/python/PythonHelper.ts

```diff
@@ -0,0 +1,110 @@
+import type { Identifier } from "@root/typeschema";
+import type { RegularTypeSchema } from "@typeschema/types.ts";
+
+export class PythonHelper {
```

Comment:
Looks like it is possible to use methods from `TypeScriptIndex`. Maybe -- we can just migrate to it from `PythonHelper`. Feel free to extend it if something looks reusable for other languages.